lms/static/sass/base/_reset.scss

```diff
@@ -9,9 +9,9 @@ html { font-size: 100%; -webkit-text-size-adjust: 100%; -ms-text-size-adjust: 10
 html, button, input, select, textarea { font-family: sans-serif; color: #222; }
 body { margin: 0; font-size: 1em; line-height: 1.4; }
 
-a { color: #00e; }
-a:visited { color: #551a8b; }
-a:hover, a:focus { color: #06e; }
+a { color: #005584; } // from the Pattern Library
```

Comment:
NIT: also, mind updating your comments to point to the exact URL of the sass where these colors are defined